geozarr-spec.md

```diff
@@ -85,44 +85,167 @@ All other CF conventions are recommended, in particular the attributes below:
+* a JSON object describing the Tile Matrix Set following the OGC standard (CamelCase!).
+
+Within the Tile Matrix Set
+* the Tile Matrix identifier for each zoom level MUST be the relative path to the Zarr group which holds the DataArray variable 
+* zoom levels MUST be provided from lowest to highest resolutions
+* the `supportedCRS` attribute of the Tile Matrix Set MUST match the crs information defined under **grid_mapping**.
+* the tile layout for each Matrix MUST correspond to the chunk layout along the two spatial dimensions listed in `_ARRAY_DIMENSIONS` of the corresponding group.
+
+
+#### Tile Matrix Set Limits
+Tile Matrix Sets may describe a larger spatial extent and more resolutions than used in the given dataset.
+In that case, users MAY specify [Tile Matrix Set Limits](https://docs.ogc.org/is/17-083r4/17-083r4.html#toc21) as described in the OGC standard to define the minimum and a maximum limits of the indices for each TileMatrix that contains actual data. However, the notation for tile matrix set does not the JSON encoding as described in the OGC standard but follows the STAC Tile Asset encoding for better readability.
+
+If used, Tile Matrix Set Limits
+* MUST list all included zoom levels
+* MAY list the min and max rows and columns for each zoom level. If omitted, it is assumed that the entire spatial extent is covered (resulting in higher chunk count of the DataArray).
+
+#### Resampling Method
+Resampling Method specifies which resampling method is used for generating multiscales. It MUST be one of the following string values. Resampling method MUST be the same across all zoom levels:
+* nearest
+* bilinear
+* cubic
+* cubic_spline
+* lanczos
+* average
+* mode
+* gauss
+* max
+* min
+* med
+* q1
+* q3
+* sum
+* rms
+
+### Multiscale examples
+#### Using Well Known Name reference
 
-* Path MUST be the relative path to the Zarr group which holds the DataArray variable 
-* Zoom levels MUST be provided from lowest to highest resolutions
-* First level path MUST reference to itself or can be omitted.
-* If the optional 'crs' attribute is missing, then the downscaled version is assumed to be non-projected (and can be displayed using a "pseudo plate-carree" projection).
+```diff
+(mandatory items in red, optional items in green)
++{
++  "multiscales":
+-       { 
+-           "tile_matrix_set": "WebMercatorQuad",
+-           "resampling_method": "nearest",
+-       }
++}
+```
+#### Using a URI
 
 ```diff
 (mandatory items in red, optional items in green)
 +{
-+  "multiscales": [
--    {
--      "name": "example",
--      "datasets": [
--        {"path": "0", "level": "0", 
-+         "crs": "EPSG:3857"},
--        {"path": "1", "level": "1",
--        {"path": "2", "level": "2"},
--        {"path": ".", "level": "3"}
--      ],
-+      "type": "gaussian",
--    }
-+  ]
++  "multiscales":
+-       { 
+-           "tile_matrix_set": "https://schemas.opengis.net/tms/2.0/json/examples/tilematrixset/WebMercatorQuad.json",
```

Comment:
I like how concise this becomes! 
I Assume this supports any TMS, e.g. one based on an equal area projection such as [epsg:6933](https://developmentseed.org/titiler/examples/code/tiler_with_custom_tms/).

Reply:
2DTMS can be registered in any registry, but the OGC registry of 2DTMS is at:

https://www.opengis.net/def/tms

which is sourced from

https://github.com/opengeospatial/2D-Tile-Matrix-Set/tree/master/registry

(the OGC API - Tiles Standard Working Group will review and consider submissions to register new 2DTMS in that repository).